app/src/app/framework/type/builder/configs/edits/ref-field-edit.component.ts

```diff
@@ -41,35 +56,75 @@ export class RefFieldEditComponent extends ConfigEditBaseComponent implements On
   public ngOnInit(): void {
     this.typeService.getTypeList().subscribe((res: CmdbType[]) => {
```

Comment:
Could you change that in the process? This is still an old remnant. At the moment **ALL** types are loaded with all information. You could use the `getTypes` method of the service and load the data in `ng-select` using lazy load.